mender-update/update_module/v3/update_module.hpp

```diff
@@ -72,8 +81,62 @@ class UpdateModule {
 	Error Cleanup();
 
 private:
+	Error PrepareStreamNextPipe();
+	Error OpenStreamNextPipe(ExpectedWriterHandler open_handler);
+	Error PrepareAndOpenStreamPipe(const string &path, ExpectedWriterHandler open_handler);
+	Error PrepareDownloadDirectory(const string &path);
+	Error DeleteStreamsFiles();
+
+	void StartDownloadProcess();
+
+	void StreamNextOpenHandler(io::ExpectedAsyncWriterPtr writer);
+	void StreamOpenHandler(io::ExpectedAsyncWriterPtr writer);
+
+	void StreamNextWriteHandler(size_t expected_n, size_t written_n, error::Error err);
+	void PayloadReadHandler(size_t n, error::Error err);
```

Comment:
Q: Could this not b econstants?

Reply:
Sure, but so could almost every variable. As long as they're not shared with anyone I think it's fine to omit const, and there's nothing inherently wrong with modifying them, sometimes it's even useful.